### PR TITLE
Add support for specifying a query to start from

### PR DESCRIPTION
**Pull request by msfjarvis**
Currently only supports fzf, can be used as `pass clip -s github` or
`pass clip --search-term github`.

I don't use rofi and am not familiar with its syntax or if it even has a similar
feature so I'll need some sort of guidance to make this available.

TODO:
- [x] Support rofi
- [ ] Maybe rename options

### CODE REVIEW

File: clip.bash [hunk 1]
```diff
@@ -81,9 +83,14 @@ cmd_clip() {
 
     # Figure out if we use fzf or rofi
     local prompt='Copy password into clipboard for 45 seconds'
-    local fzf_cmd="fzf --print-query --prompt=\"$prompt\" | tail -n1"
+    local fzf_cmd="fzf --print-query --prompt=\"$prompt\""
     local rofi_cmd="rofi -dmenu -i -p \"$prompt\""
```
Comment (ibizaman) [inline clip.bash:87]: For `rofi` you can add `--filter \"$term\"`. I tested it with:
```
echo -e "one\ntwos\nthree\ntwo" | rofi -dmenu -i -p "select" -filter "two"
```

File: clip.bash [hunk 2]
```diff
@@ -81,9 +83,14 @@ cmd_clip() {
...
 
+    if [ -n "$term" ]; then
+      fzf_cmd="$fzf_cmd -q$term"
```
Comment (ibizaman) [inline clip.bash]: This makes us support spaces in `$term`.
```suggestion
      fzf_cmd="$fzf_cmd -q$\"term\""
```

**Review by ibizaman — CHANGES_REQUESTED** [review]
Sorry for reviewing this so late. Just found the rofi option, see comments.

Thank you for this. Btw I'm curious, what would you use this option for?

**Comment by msfjarvis** [comment]
> Sorry for reviewing this so late. Just found the rofi option, see comments.
> 
> Thank you for this. Btw I'm curious, what would you use this option for?

I find it faster personally to type `pass clip -s github` then just select one of two options as opposed to letting fzf start, typing out the query and then making a selection.

File: clip.bash (hunk 1 above)
Comment (msfjarvis) [reply clip.bash:87]: Done, thanks!

**Comment by msfjarvis** [comment]
@ibizaman any other changes I should make?

**Comment by ibizaman** [comment]
Sorry about latency @msfjarvis. I wanted to test this locally first but had some issues, then no more time to test it. I’ll give it a shot this week.

**Comment by ibizaman** [comment]
Sorry for the delay. Thank you for this feature! Do you need a new release?

**Comment by msfjarvis** [comment]
Not really, I bundle the extensions into my store itself. Thanks for merging!

**Pull request merged**